src/OpenLoco/Windows/TerraForm.cpp

```diff
@@ -2070,13 +2068,13 @@ namespace OpenLoco::Ui::Windows::Terraform
             self->var_83C = wallCount;
             auto rowHover = -1;
 
-            if (_lastSelectedTree != 0xFF)
```

Comment:
this is actually a bug that I found, this should be checking the last wall option, not the last tree option. fixed now :)

Reply:
Do we think it fixes anything

Reply:
In terms of what the user perceives, no, it doesn't actually fix anything. If the last-selected-wall is not set (so it passes this check) then in the loop below there is a check if the last-selected-wall is equal to the rowInfo. No rowInfo should equal 0xFF so it wouldn't get into setting rowHover to 0xFF.

All this 'fixes' IMO is it avoids the loop, so its realistically a performance fix/increase.